llvm/test/CodeGen/CHERI-Generic/Inputs/ptradd-immediate.ll

```diff
@@ -0,0 +1,57 @@
+; RUN: llc @PURECAP_HARDFLOAT_ARGS@ < %s -o - | FileCheck %s
```

Comment:
I might be nice to also have a non-cheri run line to show how much better the integer-only codegen is.